chromium_src/components/feature_engagement/public/feature_constants.cc

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "base/feature_override.h"
+
+#include "src/components/feature_engagement/public/feature_constants.cc"
+
+namespace feature_engagement {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{kIPHPerformanceNewBadgeFeature,
```

Comment:
Note: The badge is also shown when `IsFeaturePromoActive(feature_engagement::kIPHHighEfficiencyModeFeature)`, but we don't have the promo active in Brave so I haven't overridden it. Not sure if I should or not. If we want the Promo to work, but the new badge to not show, I think it's probably better to just disable the badge in `brave_app_menu.cc`

Reply:
We typically skip `BUILDFLAG(IS_CHROMEOS)` and `BUILDFLAG(IS_FUCHSIA)` since we never build those platforms.

Reply:
Sweet, good to know! I'd just blindly copied the guard from upstream :laughing: